.gitignore

```diff
@@ -43,7 +43,6 @@ _testmain.go
 
 .DS_Store
 .idea
-.vscode
```

Comment:
Should we leave this in here and just force the commit on this PR?  Otherwise, any local customization will be a commit candidate.

Reply:
yes